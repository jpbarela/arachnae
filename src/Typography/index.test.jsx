// @flow
import * as React from "react";
import { render } from "@testing-library/react";
import { SectionHeading, Title } from "./index";

describe("Typography", () => {
  describe("SectionHeader", () => {
    it("renders", () => {
      const container = render(
        <SectionHeading>Section Heading</SectionHeading>
      );
      expect(container.baseElement.firstChild).toMatchSnapshot();
    });
  });

  describe("NumericInput", () => {
    it("renders", () => {
      const container = render(<Title>Title</Title>);
      expect(container.baseElement.firstChild).toMatchSnapshot();
    });
  });
});
