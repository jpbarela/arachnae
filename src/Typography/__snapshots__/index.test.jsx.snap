// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Typography NumericInput renders 1`] = `
<div>
  <h1>
    Title
  </h1>
</div>
`;

exports[`Typography SectionHeader renders 1`] = `
<div>
  <h2>
    Section Heading
  </h2>
</div>
`;
